admin/class-config.php

```diff
@@ -59,11 +59,14 @@ function config_page_scripts() {
 		wp_enqueue_script( 'wpseo-admin-script', plugins_url( 'js/wp-seo-admin' . WPSEO_CSSJS_SUFFIX . '.js', WPSEO_FILE ), array(
 			'jquery',
 			'jquery-ui-core',
+			'jquery-ui-progressbar',
 		), WPSEO_VERSION, true );
 		wp_localize_script( 'wpseo-admin-script', 'wpseoAdminL10n', $this->localize_admin_script() );
 		wp_enqueue_script( 'dashboard' );
 		wp_enqueue_script( 'thickbox' );
 
+		wp_enqueue_script( 'yoast-seo-content-analysis', plugins_url( 'js/dist/yoast-seo-content-analysis.min.js', WPSEO_FILE ), null, WPSEO_VERSION, true );
```

Comment:
@omarreiss Can you give me the definitive location for the recalculate button, because this location was meant to be temporary.